### PR TITLE
Trial

### PR DESCRIPTION
**Pull request by rekreider16**
Just getting caught up. Sorry I did not turn it in on time. I got the right answers though.

### CODE REVIEW

File: LoggingKata.Test/TacoParserTests.cs [hunk 1]
```diff
@@ -12,18 +12,36 @@ public void ShouldDoSomething()
...
-            // TODO: Complete Should Parse
+            // Arrange
+            TacoParser tacoParser = new TacoParser();
+
+            // Act
+            ITrackable actual = tacoParser.Parse(str);
+
+            // Assert
+            Assert.True(actual.Name.Length > 0);
+            Assert.True(actual.Location.Latitude < 181);
```
Comment (chipelmer) [inline LoggingKata.Test/TacoParserTests.cs]: This will pass if latitude is -500.  Consider getting the absolute value of the latitude and longitude and making sure they're less than 181 and 91

File: LoggingKata.Test/TacoParserTests.cs [hunk 2]
```diff
@@ -12,18 +12,36 @@ public void ShouldDoSomething()
...
+            Assert.True(actual.Location.Longitude < 91);
         }
 
         [Theory]
         [InlineData(null)]
         [InlineData("")]
```
Comment (chipelmer) [inline LoggingKata.Test/TacoParserTests.cs]: There are other edge cases that should fail parsing, such as "0,Taco Bell", "abc, xyz, Taco Bell", etc.

File: LoggingKata/Program.cs [hunk 3]
```diff
@@ -20,8 +20,47 @@ static void Main(string[] args)
...
+                        locationB = locB;
+                    }
+                }
+            }
+
+            Console.WriteLine("The two furthest Taco Bells are: ");
+            Console.WriteLine(locationA.Name);
+            Console.WriteLine(locationB.Name);
+            Console.ReadLine();
             // TODO:  Find the two Taco Bells in Alabama that are the furthest from one another.
```
Comment (chipelmer) [inline LoggingKata/Program.cs]: Remove TODOs once complete

File: LoggingKata/TacoParser.cs [hunk 4]
```diff
@@ -1,18 +1,66 @@
...
+                    Latitude = latitude
+                }
+            };
+
+            return tacoBell;
+
 
             // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            // TODO Implement
```
Comment (chipelmer) [inline LoggingKata/TacoParser.cs]: Remove TODOs once complete
